src/components/TitleCardList.tsx

```diff
@@ -152,52 +40,169 @@ const TitleCardList: React.FC<IProps> = ({
+      borderBottom={0.25}
+      borderBottomStyle="solid"
+      borderBottomColor="text"
+    >
+      {icon && (
+        <ChakraIcon
+          as={Icon}
+          name={icon}
+          _hover={{
+            path: { fill: "tranparent" },
+          }}
+        />
+      )}
       <Translation id={headerKey} />
       {isCode && (
-        <CodeBoxHeader>
-          <Red />
-          <Yellow />
-          <Green />
-        </CodeBoxHeader>
+        <Hide below="s">
+          <Flex>
+            <Box
+              width="12px"
+              height="12px"
+              bg="fail300"
+              marginRight={2}
+              borderRadius="full"
+            />
+            <Box
+              width="12px"
+              height="12px"
+              bg="gridYellow"
+              marginRight={2}
+              borderRadius="full"
+            />
+            <Box
+              width="12px"
+              height="12px"
+              bg="success300"
+              marginRight={2}
+              borderRadius="full"
+            />
+          </Flex>
+        </Hide>
       )}
-    </TableHeader>
+    </Flex>
     {content.map((listItem, idx) => {
       const { title, description, caption, link, image, alt, id } = listItem
       const isLink = !!link
+
       return isLink ? (
-        <ItemLink key={id || idx} to={link}>
-          {image && <Image image={image} alt={alt || ""} />}
-          <LeftContainer>
-            <ItemTitle>{title}</ItemTitle>
+        <LinkBox
+          key={id || idx}
+          as={Link}
+          href={link}
+          display="flex"
+          textDecoration="none"
+          hideArrow
+          justifyContent="space-between"
+          color="text"
+          marginBottom="1px"
+          padding={4}
+          width="100%"
+          boxShadow="0px 1px 1px var(--eth-colors-tableItemBoxShadow)"
+          _hover={{
+            textDecoration: "none",
+            boxShadow: "0 0 1px var(--eth-colors-primary)",
+            bg: "primary100",
+            color: "black",
+          }}
+        >
+          {image && (
+            <Box
+              as={GatsbyImage}
+              marginTop={1}
+              marginRight={4}
+              minWidth={5}
+              image={image}
+              alt={alt || ""}
+            />
+          )}
+          <Flex flex="1 1 75%" flexDirection="column" marginRight={8}>
+            <Box>{title}</Box>
 
-            <ItemDesc>{description}</ItemDesc>
-          </LeftContainer>
+            <Box fontSize="s" marginBottom="0" opacity="0.7">
+              {description}
+            </Box>
+          </Flex>
           {caption && (
-            <RightContainer>
-              <ItemDesc>{caption}</ItemDesc>
-            </RightContainer>
+            <Flex
+              flex="1 0 25%"
+              alignItems="center"
+              flexWrap="wrap"
+              marginRight={4}
+            >
+              <Box fontSize="s" marginBottom="0" opacity="0.7">
```

Comment:
Ditto with [line 138](https://github.com/ethereum/ethereum-org-website/pull/8067/files#diff-0f98fcf4d26b7b815c20d7c30103440e8ebba1dd06ef71932ef0fccd7f7bcc60R138)
```suggestion
              <Box fontSize="sm" marginBottom="0" color={descriptionCaptionColor}>
```